boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -94,6 +103,12 @@ class DcmStateMachineTest : public BaseStateMachineTest
         RESET_FAKE(turn_on_blue_led);
         RESET_FAKE(turn_on_brake_light);
         RESET_FAKE(turn_off_brake_light);
+        RESET_FAKE(turn_on_right_inverter);
+        RESET_FAKE(turn_off_right_inverter);
+        RESET_FAKE(turn_on_left_inverter);
+        RESET_FAKE(turn_off_left_inverter);
+
+        FFF_RESET_HISTORY();
```

Comment:
If we reset history here, would lines 94->109 be needed?

Reply:
I'll take FFF_RESET_HISTORY() out, seems like it's redundant

Reply:
If doing FFF_RESET_HISTORY can do the same thing as the lines of RESET_FAKE above I dont mind keeping it

Reply:
kk I'm just leaving it on there

Reply:
i meant that as like if a single FFF_RESET_HISTORY can help us get rid of the lines above then we should get rid of the lines above and keep `FFF_RESET_HISTORY();`

Reply:
tbh it's a bit unclear from the macros and the readme file so I just took FFF_RESET_HISTORY() out

Reply:
okie